timesketch/frontend-ng/src/utils/SigmaRuleTemplates.js

```diff
@@ -0,0 +1,629 @@
+/*
+Copyright 2022 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// General first part of every Sigma rule:
+const SkeletonFirst = `title: Foobar
+id: ${crypto.randomUUID()}
+description: Detects suspicious FOOBAR
+references:
+  - https://
+author: Timesketch
+date: ${new Date(Date.now()).toLocaleString('en-ZA').split(',')[0]}
+modified: ${new Date(Date.now()).toLocaleString('en-ZA').split(',')[0]}
+status: experimental
+falsepositives: unknown
+level: informational
+tags:
+    -`
+
+// General last part of every Sigma rule:
+const SkeletonLast = `falsepositives:
+    - Unknown
+status: experimental
+level: medium
+detection:
+    keywords:
+        - '*foobar*'
+    condition: keywords`
+
+// CLOUD:
+const AwsText = `${SkeletonFirst}
```

Comment:
This is repetitive - can we generate this instead?

Reply:
This is pure C&P from what is already in there: https://github.com/google/timesketch/blob/master/timesketch/frontend/src/utils/SigmaRuleTemplates.js I opt we keep it that way and make it more elegant in a separate attempt